orc8r/protos/mconfig/mconfigs.proto

```diff
@@ -92,6 +92,11 @@ message MagmaD {
 
     // List of dynamic_services
     repeated string dynamic_services = 10;
+
+    //orc8r version
+    // version should be in X.X.X format with only non-negative numbers allowed for X.
+    // A special value of 0.0.-1 will be returned, in case oc8r version is not found.
```

Comment:
Imo using 0.0.-1 as a sentinel value is overkill -- let's just not set the value, which will result in an empty string being the sentinel value. This feels more ergonomic/idiomatic for golang/protos